### ISSUE TITLE
I cant import csv file to directus in the dashboard

### ISSUE THREAD

**Issue by PedramBabakhani** [directus/directus#22585]
### Describe the Bug

I get the error of 403 when importing csv file from the side bar. I looked into it but seems like there is no solution for it.

### To Reproduce

Directus 10.10.7

### Directus Version

v10.10.7

### Hosting Strategy

Self-Hosted (Custom)

### Database

SQLite

**Comment by br41nslug** [comment]
Please provide enough information to reproduce it on our end. For example: how does the collection you're importing into look like? what are you trying to import? any errors in the browser or on the server? and such

Closing the issue for now until more information becomes available.

**Comment by PedramBabakhani** [comment]
Thanks for the quick response. 

I exported a table from another directus instance and I wanted to import it in another instance. I exported the CSV but while importing I was getting a forbidden error or in the dev mode, I saw 403 error. 

I figured it out why it was forbidden to import it. The problem was the ID in the exported CSV. I deleted the ID column in the exported CSV file and imported it again, however, it caused incremental indexing but it imported all items from the CSV file.
